test/runtests.jl

```diff
@@ -122,26 +126,26 @@ end
 
 # Empty arrays
 test_imap(
-  Any[],
```

Comment:
Well, those test was written in that way to begin with.

It's hard to say if it's in scope of Iterators but it doesn't hurt to not change the tests.